heron/tools/cli/src/python/main.py

```diff
@@ -125,7 +127,10 @@ def cleanup(files):
   :return:
   '''
   for cur_file in files:
-    shutil.rmtree(os.path.dirname(cur_file))
+    if os.path.isdir(cur_file):
+      shutil.rmtree(cur_file)
+    else:
+      shutil.rmtree(os.path.dirname(cur_file))
```

Comment:
Why add the conditional? Does `shutil.rmtree(os.path.dirname(cur_file))` fail if `cur_file` is a directory?